templates/getting-started.mdx

```diff
@@ -0,0 +1,106 @@
+---
+layout: ../../layouts/MainLayout.astro
+section: vrf
+title: "Page Title"
+whatsnext: { "Try out the advanced tutorial": "/product-path/tutorials/advanced" }
+---
+
+{/\*
+Fill out the fields in the frontmatter above.
+
+- Section: What section of the top-nav this page appears in gettingStarted, dataFeeds, vrf, automation, chainlinkFunctions, anyApi, nodeOperator
+- Title: The title appearing at the top of the page
+- Whats Next: A list of pages to include in the What's Next section. Preferably imperative sentences describing the goal of the next page.
+  \*/}
+
+{/* Imports if necessary */}
+import { Aside } from "@components"
+
+{/* Add an overview description of the guide. What will the reader do and why is it important to try it? Getting started guides show the most basic way to use a technology or application. */}
+To learn how this technology works, set up your environment and run a quick example.
+
+{/_
+Optional - Use the <Aside> component to call extra attention to risks, warnings, or potential issues.
+Available types are:
+note
+tip
+caution
+danger
+_/}
+
+<Aside type="note" title="Example notice">
+  Example notice.
+</Aside>
+
+{/* Optional - A prerequisites section with an unordered list of what you need before you begin the guide. */}
+
+## Before you begin
+
+Before you start this guide, you need the following items:
+
+- Item 1
+- Item 2
+- Item 3
+
+{/* Heading names with ## <h2> style. If <h3> or <h4> are required for sub-sections, add more # characters. */}
+
+## Setup steps
+
+{/* Brief description of what components you are setting up and why it is necessary for the guide. */}
+Run `./setup-example` to install the required config files.
+
+{/* An ordered list of steps to run with example output and descriptions of each step where possible. */}
+
+1. Run the `./setup-example` command.
+
+   {/* Command to run for setting up */}
+
+   ```shell
+   ./setup-example
+   ```
+
+   This command prints the expected result.
+
+   {/* Command output */}
+
+   ```text
+   Expected results
+   ```
+
+1. {/* Step 2 */}
+
+   1. {/* Sub-step A */}
+   1. {/* Sub-step B */}
+
+1. {/* Step 3 */}
+
```

Comment:
Do you think we should include an example of a Markdown table? (I'll admit, I just really like tables.)

Reply:
Applied here: https://github.com/smartcontractkit/documentation/pull/1425/files